src/GraphMetadataViewExtension/GraphMetadataViewModel.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.IO;
+using System.Windows.Media.Imaging;
+using Dynamo.Core;
+using Dynamo.Graph.Workspaces;
+using Dynamo.GraphMetadata.Controls;
+using Dynamo.UI.Commands;
+using Dynamo.Wpf.Extensions;
+
+namespace Dynamo.GraphMetadata
+{
+    public class GraphMetadataViewModel : NotificationObject
+    {
+        private readonly ViewLoadedParams viewLoadedParams;
+        private HomeWorkspaceModel currentWorkspace;
+
+        public DelegateCommand AddCustomPropertyCommand { get; set; }
+
+        public string GraphDescription
+        {
+            get { return currentWorkspace.Description; }
+            set { currentWorkspace.Description = value; RaisePropertyChanged(nameof(GraphDescription)); }
+        }
+
+        public string GraphAuthor
+        {
+            get { return currentWorkspace.Author; }
+            set { currentWorkspace.Author = value; RaisePropertyChanged(nameof(GraphAuthor)); }
+        }
+
+        public Uri HelpLink
+        {
+            get { return currentWorkspace.GraphDocumentationURL; }
+            set { currentWorkspace.GraphDocumentationURL = value; RaisePropertyChanged(nameof(HelpLink)); }
+        }
+
+        public BitmapImage Thumbnail
+        {
```

Comment:
For these image convertors function should we try to guard them better ? And if there is an exception we can show something in the console but allow to program to run.